CMSgov/mint-app
# [EASI-3553] Update non claims based payment options (Migration Hotfix)

# EASI-3553

## Changes and Description

- Extending the original PR for this branch, this includes logic in the migration to remove GRANTS entries as a safeguard



## How to test this change

- Move the corresponding migration to another folder
- Bring up fresh docker containers
- Open a row in plan_payments and add {GRANTS, OTHER} as non_claims_payment entries
- Move the migration back into the migrations folder
- Run scripts/dev db:migrate
- Observe that the corresponding data now shows only {OTHER}

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
